### ISSUE TITLE
Display help and list of available commands

### ISSUE THREAD

**Issue by nochso** [phparsenal/fast-forward#8]
To display a list of available commands and short descriptions:
`ff help`

Display for command: argument list, detailed description with usage examples
`ff help <command>`

CLImate already creates a nice argument list:
```
fast-forward 0.1

Usage: /home/amblin/dev/web/fast-forward/cli-launch.php [-c cmd] [-d desc] [-s shortcut]

Required Arguments:
	-c cmd
		Command to be saved
	-s shortcut
		Shortcut or alias used for searching

Optional Arguments:
	-d desc
		Short description of the command
```
However the Usage line should be more like this:
```
Usage: ff add [-c cmd] [-d desc] [-s shortcut]
Usage: ff [-c cmd] [-d desc] [-s shortcut] add
```
Path should be `ff` and the command should be displayed as shown. Any takers?

**Comment by nochso** [comment]
Trying to fix the usage line is harder than I thought. I might just fork CLImate for now and send PRs as it becomes stable and sensible.

Basically handling of commands without prefixes needs to be addressed. The "add" part of the command should be required to have the value "add" so that $cli->arguments->parse() will match desired behaviour.

**Comment by nochso** [comment]
Implemented in #30
